lib/generators/hyrax/templates/config/initializers/hyrax.rb

```diff
@@ -219,6 +219,9 @@
   # How long to hold the lock in milliseconds
   # config.lock_time_to_live = 60_000
 
+  # Maximum number of collections possible in solr queries where we expect to return all collections
+  # config.collection_query_limit = Collection.count + 1_000
+
```

Comment:
I'm not sure I follow why Collection.count + 1_000.  Why not just 1_000?  Or the greater of Collection.count or 1_000?

Reply:
@julesies and @cam156 said to use the count of collections plus 1000 so that's what i did.

Reply:
👍